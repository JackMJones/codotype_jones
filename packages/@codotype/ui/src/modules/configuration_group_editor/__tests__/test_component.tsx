// import * as React from "react";
// import TestRenderer from "react-test-renderer";
import {
  OptionValue,
  ComponentBuilderConfigurationGroup,
  buildConfigurationGroupValue
} from "../component";

describe("ConfigurationGroupEditor", () => {
  test("render", () => {
    const configurationGroupValue: OptionValue = {
      [ComponentBuilderConfigurationGroup.identifier]: buildConfigurationGroupValue(
        ComponentBuilderConfigurationGroup.properties
      )
    };

    console.log(configurationGroupValue);

    expect(configurationGroupValue).toMatchSnapshot();
  });
});
