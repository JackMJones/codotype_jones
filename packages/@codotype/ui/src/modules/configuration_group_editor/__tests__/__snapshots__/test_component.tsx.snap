// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfigurationGroupEditor render 1`] = `
Object {
  "components_group": Object {
    "components": Array [
      Object {
        "componentName": "",
        "componentSlug": "",
        "props": Array [
          Object {
            "desc": "",
            "name": "",
            "type": "",
          },
        ],
        "tests": Object {
          "enabled": true,
          "value": Object {
            "testType": "",
          },
        },
      },
    ],
  },
}
`;
